civis/futures.py

```diff
@@ -221,6 +221,25 @@ def _poll_and_set_api_result(self):
             except Exception as e:
                 self._set_api_exception(exc=e)
 
+    def outputs(self):
+        """
+        Block on job completion and return a list of run outputs if the
```

Comment:
Sorry to be nitpicky, but I think it'd be more in line with [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#sections) to have this sentence be shorter and fit on the first line like it was initially (`"""Block on job completion and return a list of run outputs.`). I think the "if the job is successful" part could be a separate sentence (see "Extended Summary" in the numpydoc page).